tests/indent/go/multiple-vars.go

```diff
@@ -0,0 +1,19 @@
+package main
+
+var (
+	thing    = 1
+	thingTwo = 2
+) // <-- This paren should be at 0 instead of indented
+
+var (
+	thing    = 1
+	thingTwo = 2
+)
+
+func main() {
+	// It should be
+	var (
+		thing    = 1
+		thingTwo = 2
+	)
+}
```

Comment:
this was on my PC from an old issue. essentially the same problem. I can't remember which issue it was.